sourcegraph/learn
# Fix a few typos and small problems on the code search cheat sheet page

I was just reading the page to learn about the search syntax, spotted a few problems, and decided to fix them to improve this page.

For the types, I used https://docs.sourcegraph.com/code_search/reference/queries as a source.

## Type of change
Typo fixes.

### What should this PR do?
There's no related Asana or Jira issue. These are random minor fixes to a page content, not requested by anyone.

### Why are we making this change?
I intend to improve this page.

### What are the acceptance criteria? 
The page should not look broken. The content should be better than it was before the change.

### How should this PR be tested?
- Check out the GitHub diff (I think the two-column view is best for this), see if you spot any typos in my version.
- If there's a test environment to run this branch then go to that environment, scroll through the page and see the new content, and make sure it looks good.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
